test/system/040-serve.bats

```diff
@@ -84,14 +83,14 @@ verify_begin="podman run --rm -i --label \"RAMALAMA container\" --security-opt=l
 
     run_ramalama serve -p ${port1} --detach ${model}
     cid="$output"
-    run podman wait --condition=running $cid
+    run -0 podman wait --condition=running $cid
```

Comment:
FYI, this seems rather pointless if you do `podman run -d` followed by that. The container will always be running here unless it immediately exited but even then the wait would just hang.

Reply:
Wouldn't there be a starting stage before Running?

Reply:
Well in podman container startup logic sure but a `run -d` will only return after the container state has been changed to running (but well that just means we exec'ed the pid 1 in the container so that does not necessary mean that the container process is ready to do whatever it want).

Reply:
Yup, but makes the check `podman wait --condition running` rather useless.